conftest.py

```diff
@@ -26,3 +29,16 @@ def court():
 def court_ball_game(court):
     court_ball_game = CourtBallGame.objects.create(court=court, ball_game=BallGame.Basketball)
     return court_ball_game
+
+
+@pytest.fixture
+def game_event(court):
+    return GameEvent.objects.create(id=2, time=timezone.now(), level_of_game=3,
+                                    min_number_of_players=2, max_number_of_players=5,
+                                    court=court, ball_game=BallGame.Baseball)
+
+
+@pytest.fixture
+def game_event_player(game_event, player):
+    return GameEventPlayer.objects.create(game_event=game_event, player=player,
+                                          ball_responsible=False)
```

Comment:
since #93 got merged thoes fixtures already defined in the conftest, so you can do quick rebase and make use of thoes